include/tcframe/spec/variable/Variable.hpp

```diff
@@ -76,6 +76,11 @@ class Variable {
             throw runtime_error("Cannot parse for " + context + ". Found: '" + found + "'");
         }
     }
+
+    static void parseRawLine(istream* in, string& val) {
+        getline(*in, val);
+        in->unget();
```

Comment:
If the getline reach EOF instead of '\n', then the last character will be unget?

Like if input = "hello" (no newline), then 'o' will be unget?

I'm not sure whether something can go wrong because of this though